exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/ImportGrabber.java

```diff
@@ -63,9 +62,21 @@ public void traverseStaticImportOnDemandDeclaration(StaticImportOnDemandDeclarat
 
   }
 
-  public static List<String> getMethods(Java.CompilationUnit cu){
-    ImportGrabber visitor = new ImportGrabber();
-    cu.getPackageMemberTypeDeclarations()[0].accept(visitor.finder.comprehensiveVisitor());
+  /**
+   * Creates list of imports that are present in compilation unit.
+   * For example:
+   * [import io.netty.buffer.DrillBuf;, import org.apache.drill.exec.expr.DrillSimpleFunc;]
+   *
+   * @param compilationUnit compilation unit
+   * @return list of imports
+   */
+  public static List<String> getImports(Java.CompilationUnit compilationUnit){
```

Comment:
When writing code for UDFs, we recommend users to use the fully qualified class names instead of imports.
What if we enhance our UDFs generation code to use imports from function class? This way, fully qualified class names usage won't be required. Since I thought it could be used for future enhancement, I have left `ImportGrabber`.